src/index.js

```diff
@@ -1,21 +1,17 @@
 "use strict";
 
-let WritableStream = require("./writable_stream");
+const WritableStream = require("./writable_stream");
```

Comment:
This still seems odd to me, as the community in my opinion strives for the latter. And especially while I don't want such variables to be reassigned later on `let` seems not right to me. In my opinion `let` gives a very clear signal, that I want to change the value after the first assignment. Using it by default removes that signal-function. But maybe its another discussion.